project/project.go

```diff
@@ -42,6 +44,10 @@ type Descriptor struct {
 	Metadata map[string]interface{} `toml:"metadata"`
 }
 
+func (d *Descriptor) ProjectMetadata() platform.ProjectMetadata {
+	return platform.ProjectMetadata{}
+}
+
```

Comment:
As much as I like it, these lines won't be needed in your final implementation, correct?

Reply:
Nope, we can do without. Maybe I can comment this out,

Reply:
If something isn't necessary, we delete it, not comment out. 😁